Kitodo/src/main/java/org/kitodo/forms/BaseForm.java

```diff
@@ -43,6 +43,7 @@
     protected static final String ERROR_READING = "errorReading";
     protected static final String ERROR_RELOADING = "errorReloading";
     protected static final String ERROR_SAVING = "errorSaving";
+    protected static final String ERROR_DELETING_LDAP_GROUPE = "ldapGroupInUse";
```

Comment:
It can be placed in LdapGroupForm, as it is not going to be used in other forms.

Reply:
Done ;)